rules/linux/process_creation/proc_creation_lnx_userdel.yml

```diff
@@ -0,0 +1,24 @@
+title: Deleting an Account or a Group in CLI
+id: 8a46f16c-8c4c-82d1-b121-0fdd3ba70a84
+status: experimental
+description: Detects usage of the "userdel" or "groupdel" binary to delete user or delete group
+references:
+    - https://cve.mitre.org/cgi-bin/cvename.cgi?name=2019-19882
```

Comment:
```suggestion
```
Did not find a APT or malware report using them. This command are use to remove the trace of these ...

Reply:
I deleted old ref and update new ref